Module.php

```diff
@@ -144,8 +144,9 @@ public function bootstrap($app)
         $app->on(Application::EVENT_BEFORE_REQUEST, function () use ($app) {
             $app->getView()->on(View::EVENT_END_BODY, [$this, 'renderToolbar']);
         });
-
-        $app->getUrlManager()->addRules([
+        $urlManager = $app->getUrlManager();
+        $urlManager->ruleConfig['class'] = 'yii\web\UrlRule';
```

Comment:
Ok, I got it.

Maybe, it's better to save current `ruleConfig['class']` to the temp variable, set default, add the debug rules and restore the original `ruleConfig['class']`?